### DISCUSSION TITLE
Como me preparar para google kickstart?

### DISCUSSION THREAD

**Discussion by rafgam10** [frontendbr/forum#2326] [Perguntas e Respostas]
Olá, quero sabe como eu faço para me preparar para google kickstart, eu só sei python, html e css, quero muito participar mais quero me sair bem, tenho que aprender C++?, posso usar python?, o que estudar com python para isso? e dicas.

**Comment by felquis** [comment]
Não sei responder sua pergunta, mas não quero te deixar no vacuo, então vou te devolver uma pergunta, o que é o google kickstart?

**Reply by rafgam10** [reply]
O Google Kick Start é uma competição global de codificação online organizada pelo Google para aqueles que estão interessados ​​em resolver problemas algorítmicos divertidos e desafiadores

**Reply by felquis** [reply]
eu acho que a google oferece conteúdo para quem quer se preparar para este tipo de desafio

**Reply by rafgam10** [reply]
porque sou novo e eu não sei a onde posso começar e não tem muito conteundo br falando disso

**Comment by tompereira09** [comment]
Tambem queria participar, começe por estudar algoritmos e data structures

**Comment by igorlvicente** [comment]
Recomendo esse livro. É considerado a "bíblia para os competidores".
https://www.amazon.com.br/Competitive-Programming-4-Book-1/dp/B093G64LY2


_Não é propaganda. Estou ganhando nada por isso._

**Reply by tompereira09** [reply]
Obrigado Igor. Estou procurando um pdf gratuito do livro para começar a ler.
